components/script/dom/htmltextareaelement.rs

```diff
@@ -64,11 +64,14 @@ impl LayoutHTMLTextAreaElementHelpers for LayoutDom<HTMLTextAreaElement> {
     #[allow(unsafe_code)]
     unsafe fn get_value_for_layout(self) -> String {
```

Comment:
feel free to rename this to `value_for_layout` for consistency with `<input>`, if you want.